tests/testthat/test-run.R

```diff
@@ -36,6 +36,14 @@ test_that("run_one", {
   wipe_results()
 })
 
+test_that("get_params_summary returns a tibble",{
+  bm <- run_benchmark(placebo, cpu_count = 1, output_type = "message")
+
+  expect_s3_class(get_params_summary(bm), "tbl_df")
```

Comment:
It would be nice to assert that this includes the info we think it should as well

Reply:
Excellent suggestion